### PR TITLE
Add Katy & Robert as maintainers

### PR DESCRIPTION
**Pull request by gonuke**
Adding @katyhuff and @FlanFlanagan as maintainers

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -78,3 +78,5 @@ extra:
     - scopatz
     - gonuke
     - bam241
+    - katyhuff
+    - FlanFlanagan
```
Comment (scopatz) [inline recipe/meta.yaml]: Same

Comment (FlanFlanagan) [reply recipe/meta.yaml]: yes

Comment (katyhuff) [reply recipe/meta.yaml]: yes

**Comment by gonuke** [comment]
This did not generate a list of maintainers in the `README` - and it appears there never was one.  Is there something we need to do to make that happen? @scopatz

**Pull request merged**